### PR TITLE
[4.0] AfterSave events to return void

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

The return values of after save events are not used. Unless they should be, this changes them to be void.

### Testing Instructions

Code review or try saving stuff. Should work like before.

### Documentation Changes Required

No.

### CODE REVIEW

File: plugins/system/privacyconsent/privacyconsent.php [hunk 1]
```diff
@@ -152,11 +152,11 @@ public function onUserBeforeSave($user, $isNew, $data)
 	 * @param   boolean  $result  true if saving the user worked
 	 * @param   string   $error   error message
 	 *
-	 * @return  boolean
+	 * @return  void
 	 *
 	 * @since   3.9.0
 	 */
-	public function onUserAfterSave($data, $isNew, $result, $error)
+	public function onUserAfterSave($data, $isNew, $result, $error): void
```
Comment (Quy) [inline plugins/system/privacyconsent/privacyconsent.php:159]: Update `return`s below.

File: plugins/system/fields/fields.php [hunk 2]
```diff
@@ -193,13 +191,13 @@ public function onUserAfterSave($userData, $isNew, $success, $msg)
...
 		{
-			return true;
+			return;
 		}
 
 		// Trigger the events with a real user
 		$this->onContentAfterSave('com_users.user', $user, false, $userData);
 
-		return true;
+		return;
```
Comment (Quy) [inline plugins/system/fields/fields.php]: Delete?

Comment (SharkyKZ) [reply plugins/system/fields/fields.php]: Yes.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 26740dfa718e584fc258d1bde9314e535824a24c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29146">issues.joomla.org/tracker/joomla-cms/29146</a>.</sub>

**Comment by bonzani** [comment]
I have tested this item :white_check_mark: successfully on 26740dfa718e584fc258d1bde9314e535824a24c<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29146">issues.joomla.org/tracker/joomla-cms/29146</a>.</sub>

**Comment by SharkyKZ** [comment]
RTC.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/29146">issues.joomla.org/tracker/joomla-cms/29146</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**